go/externalstest/test_common.go

```diff
@@ -3,9 +3,10 @@
 
 // +build !production
 
-package externals
+package externalstest
```

Comment:
Let me know if there's a better name I should use.

Reply:
Thank you @akalin-keybase i appreciate your help.